### PR TITLE
fix: suppress warnings for invalid url source

### PR DESCRIPTION
**Pull request by raki-git**
suppress warnings for invalid url source and removing centos7 images as we won't be using it anymore

```
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ rpmlint seiscomp/
=========================================================================================================== rpmlint session starts ===========================================================================================================
rpmlint: 2.5.0
configuration:
    /usr/lib/python3.12/site-packages/rpmlint/configdefaults.toml
    /etc/xdg/rpmlint/fedora-legacy-licenses.toml
    /etc/xdg/rpmlint/fedora-spdx-licenses.toml
    /etc/xdg/rpmlint/fedora.toml
    /etc/xdg/rpmlint/scoring.toml
    /etc/xdg/rpmlint/users-groups.toml
    /etc/xdg/rpmlint/warn-on-functions.toml
checks: 32, packages: 1

seiscomp.spec: W: invalid-url Source0: seiscomp-6.3.1-rhel-8-x86_64.tar.gz
====================================================================== 0 packages and 1 specfiles checked; 0 errors, 1 warnings, 0 filtered, 0 badness; has taken 0.1 s ======================================================================
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ 
```

```
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ rpmlint caps-plugins/
=========================================================================================================== rpmlint session starts ===========================================================================================================
rpmlint: 2.5.0
configuration:
    /usr/lib/python3.12/site-packages/rpmlint/configdefaults.toml
    /etc/xdg/rpmlint/fedora-legacy-licenses.toml
    /etc/xdg/rpmlint/fedora-spdx-licenses.toml
    /etc/xdg/rpmlint/fedora.toml
    /etc/xdg/rpmlint/scoring.toml
    /etc/xdg/rpmlint/users-groups.toml
    /etc/xdg/rpmlint/warn-on-functions.toml
checks: 32, packages: 1

caps-plugins.spec: W: invalid-url Source0: caps-plugins-2024.066-seiscomp-6-rhel-8-x86_64.tar.gz
====================================================================== 0 packages and 1 specfiles checked; 0 errors, 1 warnings, 0 filtered, 0 badness; has taken 0.1 s ======================================================================
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ 
```

linked to https://github.com/GeoNet/packaging/pull/367

### CODE REVIEW

**Comment by raki-git** [comment]
Test outcome

```
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ cat /etc/xdg/rpmlint/rpmlintrc.toml
#suppress warnings for invalid-url Source

Filters = ['W: invalid-url Source*']
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$
```

```
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ rpmlint  seiscomp/
=========================================================================================================== rpmlint session starts ===========================================================================================================
rpmlint: 2.5.0
configuration:
    /usr/lib/python3.12/site-packages/rpmlint/configdefaults.toml
    /etc/xdg/rpmlint/fedora-legacy-licenses.toml
    /etc/xdg/rpmlint/fedora-spdx-licenses.toml
    /etc/xdg/rpmlint/fedora.toml
    /etc/xdg/rpmlint/rpmlintrc.toml
    /etc/xdg/rpmlint/scoring.toml
    /etc/xdg/rpmlint/users-groups.toml
    /etc/xdg/rpmlint/warn-on-functions.toml
checks: 32, packages: 1

====================================================================== 0 packages and 1 specfiles checked; 0 errors, 0 warnings, 1 filtered, 0 badness; has taken 0.1 s ======================================================================
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ 
```

```
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ rpmlint caps-plugins/
=========================================================================================================== rpmlint session starts ===========================================================================================================
rpmlint: 2.5.0
configuration:
    /usr/lib/python3.12/site-packages/rpmlint/configdefaults.toml
    /etc/xdg/rpmlint/fedora-legacy-licenses.toml
    /etc/xdg/rpmlint/fedora-spdx-licenses.toml
    /etc/xdg/rpmlint/fedora.toml
    /etc/xdg/rpmlint/rpmlintrc.toml
    /etc/xdg/rpmlint/scoring.toml
    /etc/xdg/rpmlint/users-groups.toml
    /etc/xdg/rpmlint/warn-on-functions.toml
checks: 32, packages: 1

====================================================================== 0 packages and 1 specfiles checked; 0 errors, 0 warnings, 1 filtered, 0 badness; has taken 0.1 s ======================================================================
[rakid@hutl24194:~/GEO_Workspae/packaging/inputs/el8]$ 
```

**Review by ardrigh — APPROVED** [review]
New configuration file is very useful

File: images/rpmbuild-centos7/rpmlintrc.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#suppress warnings for invalid-url Source0
+
+Filters = ['W: invalid-url Source*']
```
Comment (Mossman1215) [inline images/rpmbuild-centos7/rpmlintrc.toml:3]: we won't be keeping centos7 images now and this should have been removed already

Comment (raki-git) [reply images/rpmbuild-centos7/rpmlintrc.toml]: Thanks @Mossman1215 just removed centos7 image

**Review by Mossman1215 — APPROVED** [review]
looking good.
let's see if it resolves the linter implementation

**Pull request merged**